app/config/ErrorHandler.scala

```diff
@@ -28,5 +29,9 @@ class ErrorHandler @Inject()(val messagesApi: MessagesApi, implicit val appConfi
   override def standardErrorTemplate(pageTitle: String, heading: String, message: String)(implicit request: Request[_]): Html =
     views.html.errors.standardError(appConfig, pageTitle, heading, message)
 
-  def showInternalServerError(implicit request: Request[_]): Result = InternalServerError(internalServerErrorTemplate)
+  def showInternalServerError(implicit request: Request[_]): Result = InternalServerError(standardErrorTemplate(
+    Messages("standardError.title"),
```

Comment:
Can we use the already imported MessagesApi instead of importing Messages for this?